### PR TITLE
2 minute Lock

### PR DESCRIPTION
**Pull request by MonsieurPetion**
2 minute Lock @lancelafontaine

### CODE REVIEW

File: app/templates/add.html [hunk 1]
```diff
@@ -239,7 +238,7 @@
             alert("You Ran Out Of Time!");
             $(".close").trigger('click');
           }
-        }, 1000);
+        }, 1000);*/
```
Comment (Taimoorrana1) [inline app/templates/add.html]: We should get rid of commented code

File: app/templates/index.html [hunk 2]
```diff
@@ -209,45 +209,14 @@
         $('div.number').click(function() {
             var number = parseInt($('.number').text());
         });
+        /* 2 minute lock
```
Comment (Taimoorrana1) [inline app/templates/index.html]: We should get rid of commented code

File: app/templates/index.html [hunk 3]
```diff
@@ -209,45 +209,14 @@
...
-                if(m == 0 && s == 0 && $(".modal").hasClass("in"))
-                {
-                    $(".defaultCountdown").html("00 : 00");
-                    alert("You Ran Out Of Time!");
-                    $(".close").trigger('click');
-                }
-            }, 1000);
-        });
+
         $(function(){
```
Comment (Taimoorrana1) [inline app/templates/index.html]: We should get rid of commented code

File: app/templates/modify.html [hunk 4]
```diff
@@ -87,45 +87,6 @@
...
-                if(m == 0 && s == 0 && $(".modal").hasClass("in"))
-                {
-                    $(".defaultCountdown").html("00 : 00");
-                    alert("You Ran Out Of Time!");
-                    $(".close").trigger('click');
-                }
-            }, 1000);
-        });
 
     //$(window).load(function(){
```
Comment (Taimoorrana1) [inline app/templates/modify.html]: We should get rid of commented code

File: app/templates/add.html (hunk 1 above)
Comment (MonsieurPetion) [inline app/templates/add.html]: ok

File: app/templates/index.html (hunk 2 above)
Comment (MonsieurPetion) [inline app/templates/index.html]: ok

File: app/templates/index.html (hunk 3 above)
Comment (MonsieurPetion) [inline app/templates/index.html]: ok

File: app/templates/modify.html (hunk 4 above)
Comment (MonsieurPetion) [inline app/templates/modify.html]: ok

**Comment by MonsieurPetion** [comment]
done

File: app/templates/add.html [hunk 5]
```diff
@@ -210,36 +209,10 @@
...
-          {
-            $(".defaultCountdown").html("00 : 00");
-            alert("You Ran Out Of Time!");
-            $(".close").trigger('click');
-          }
-        }, 1000);
+<<<<<<< HEAD
+=======
+
+>>>>>>> master
```
Comment (arekmano) [inline app/templates/add.html]: Missed merge conflict artifact

**Comment by MonsieurPetion** [comment]
should be ok now

File: app/templates/modify.html (hunk 4 above)
Comment (lancelafontaine) [inline app/templates/modify.html]: Still some commented code here

Comment (MonsieurPetion) [inline app/templates/modify.html]: Ahh This commented code was from other team!Will remove now

**Comment by arekmano** [comment]
This pull request is useless. We will throw the front-end code out anyways.

**Pull request closed without merging**